### ISSUE TITLE
Getting it to build feedback

### ISSUE THREAD

**Issue by Arielore** [ILikesCaviar/BDO_Enhancement_Tool#10]
Hi,

I am pretty exited to get to try this program! I struggled to get the environment just right to build it so I attached the Yaml file that anaconda made to help others to skip that fun. 

I had two minor problems after following the readme; 
  1. build_run.py line#60 fails the build as the folder 'QtCommon/forms' is not included with the source. I just made an empty folder and it was happy.
  2. One additional dependency had to be installed 'pyqtgraph' that wasn't on the readme list.

My yaml anaconda environment for win10 in PyCharm:
[environment.txt](https://github.com/ILikesCaviar/BDO_Enhancement_Tool/files/6330014/environment.txt)

Thanks for your efforts!

**Comment by ILikesCaviar** [comment]
Hey, thanks for giving feedback. I added pyqtgraph to the requirements and I believe I fixed the build error a while ago. I'm going to go ahead and push to the code base even though there are some experimental things I'm working on. I probably should have branched but whatever. Pull and see if that helps.

**Comment by Arielore** [comment]
Sorry to bug you about small things. I am still a bit chicken to try and pull changes into public code. I’ll test the new one and close the issue if it needs me to. It’s a pretty neat platform for analysis coding. I may fork and try to make some features in the future as I learn more about AI at school. Maybe even do some cooking/node planning. Have a good one!Mike Valliant  Sent from Mail for Windows 10 From: Ryan McConnellSent: Saturday, April 17, 2021 2:48 PMTo: ILikesCaviar/BDO_Enhancement_ToolCc: Arielore; AuthorSubject: Re: [ILikesCaviar/BDO_Enhancement_Tool] Getting it to build feedback (#10) Hey, thanks for giving feedback. I added pyqtgraph to the requirements and I believe I fixed the build error a while ago. I'm going to go ahead and push to the code base even though there are some experimental things I'm working on. I probably should have branched but whatever. Pull and see if that helps.—You are receiving this because you authored the thread.Reply to this email directly, view it on GitHub, or unsubscribe.
